### ISSUE TITLE
KeyError on linode-cli volumes delete

### ISSUE THREAD

**Issue by tamalsaha** [linode/linode-cli#239]
I am seeing the following error when trying to delete an unattached volume. Can you please confirm if this is a bug in the cli or I am doing something wrong. Also, the delete operation fails as a result. Volume list operation works fine.

```
$linode-cli volumes delete 122602

Traceback (most recent call last):
  File "/usr/lib/python3.6/configparser.py", line 846, in items
    d.update(self._sections[section])
KeyError: ''

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/tamal/.local/bin/linode-cli", line 8, in <module>
    sys.exit(main())
  File "/home/tamal/.local/lib/python3.6/site-packages/linodecli/__init__.py", line 461, in main
    cli.handle_command(parsed.command, parsed.action, args)
  File "/home/tamal/.local/lib/python3.6/site-packages/linodecli/cli.py", line 560, in handle_command
    result = self.do_request(operation, args)
  File "/home/tamal/.local/lib/python3.6/site-packages/linodecli/cli.py", line 434, in do_request
    parsed_args = self.config.update(parsed_args)
  File "/home/tamal/.local/lib/python3.6/site-packages/linodecli/configuration.py", line 136, in update
    return self.update_namespace(namespace, dict(self.config.items(username)))
  File "/usr/lib/python3.6/configparser.py", line 849, in items
    raise NoSectionError(section)
configparser.NoSectionError: No section: ''
```

**Comment by Dorthu** [comment]
After some fiddling, I was able to reproduce this by:

* Using the CLI with a token set in the environment (using the `LINODE_CLI_TOKEN` shell variable)
* Removing all config files for the CLI (both `~/.config/linode-cli` and `~/.linode-cli` do not exist)

Does that look like your setup?

Either way I'll put up a fix for the bug I found shortly, but if that isn't your setup I'll try to find another way to cause it to occur.

As a workaround, you can run `linode-cli configure` to populate a CLI config file that should avoid this issue.  If this isn't possible, the fix should be available shortly.

**Comment by tamalsaha** [comment]
Hi,
My set up looks like your first bullet point. I am trying to run this in GH
actions. So, I installed it using pip and tried to auth from the env
variable without any manual interaction.

Thanks.

**Comment by Dorthu** [comment]
Excellent, the above fix should get it sorted out then.  I'll have that merged/deployed by this afternoon.

**Comment by Dorthu** [comment]
This should be fixed in 5.1.0 which was just released
